onnxruntime/contrib_ops/cuda/fused_conv.cc

```diff
@@ -80,12 +82,13 @@
                                                     Base::s_.y_tensor,
                                                     Base::s_.y_data));
       if (has_b) {
-        CUDNN_RETURN_IF_ERROR(cudnnAddTensor(Base::CudnnHandle(), &alpha, Base::s_.b_tensor, Base::s_.b_data,
-                                             &alpha, Base::s_.y_tensor, Base::s_.y_data));
+        //Base::s_.b_tensor
+        has_b = cudnnStatus_t::CUDNN_STATUS_SUCCESS != cudnnAddTensor(Base::CudnnHandle(), &alpha, Base::s_.b_tensor, Base::s_.b_data,
+                                                                      &alpha, Base::s_.y_tensor, Base::s_.y_data);
```

Comment:
It looks like we are swallowing an error. It is especially important to retain and return this error because we are supposedly fixing the error and we are interested why else it would fail.

Reply:
Reordered logic to directly report the error on call failure.